### PR TITLE
add UEFN Windows testing builds and missing info on some other builds

### PR DESCRIPTION
**Pull request by AllyJaxx**

### CODE REVIEW

**Comment by 3ldor** [comment]
This repository is for public release builds, remove the testing builds or open a new MR that only contains the info for 23.00 and 27.11.

**Comment by AllyJaxx** [comment]
Updated accordingly, wasn't aware that this was specifically public releases.

File: README.md [hunk 1]
```diff
@@ -390,8 +390,8 @@ If you think something is wrong or if you would like to add something, open an i
 
 # Season 23
 | Build version     | Engine version | Net CL | Build date | Manifest                       |
-| ----------------- | -------------- | ------ | ---------- | ------------------------------ |
-| 23.00-CL-23344627 |                |        |            | rI9ROK-1oURsD0RM6UqYcNk1sArzAw |
+| ------------------| -------------- | ------ | ---------- | ------------------------------ |
```
Comment (3ldor) [inline README.md]:
```suggestion
| ----------------- | -------------- | ------ | ---------- | ------------------------------ |
```

**Comment by AllyJaxx** [comment]
not quite sure how I managed to miss that, fixed.

**Comment by 3ldor** [comment]
Thanks!

**Pull request merged**